app/controllers/conference_registrations_controller.rb

```diff
@@ -20,6 +20,11 @@ def new
       redirect_to edit_conference_conference_registrations_path(@conference.short_title)
     end
 
+    if @conference.registration_limit_exceeded?
```

Comment:
Sorry but in conference_registrations_controller I'm not updating the conference object, so I need to check it the limit has exceeded before show the registration form. Maybe there are another way to do it that for sure I don't know :)